dec comm call announcement (#83)
ropensci/roweb3: content/commcalls/2020-12-16/index.md

author:
  - Kara Woo
  - Daniella Lowenberg
  - Matt Jones
  - Carl Boettiger
  - Karthik Ram
organizer: Stefanie Butland
location: Community call (teleconference)
country: "\U0001F310"
attendees: All are welcome
slug: dec2020-datarepos
notes: #link to collaborative notes
outputs:
- HTML
- Calendar
# vimeo:
#  - id: '378010209'
resources:
  - url: https://github.com/ropensci-org/community-calls/issues/9
    name: Discussion on GitHub
    details: Read the thread that led to this Community Call
  - url: https://arxiv.org/abs/2002.11626
    name: A Realistic Guide to Making Data Available Alongside Code to Improve Reproducibility
    details: arXiv preprint by Nicholas Tierney, Karthik Ram
  - url: https://doi.org/10.1177/2053951719836258
    name: Enforcing public data archiving policies in academic publishing. A study of ecology journals
    details: Sholler D, Ram K, Boettiger C, Katz DS (2019) Big Data & Society.
  - url: https://doi.org/10.1371/journal.pone.0230416
    name: The citation advantage of linking publications to research data
    details: Colavizza G, Hrynaszkiewicz I, Staden I, Whitaker K, McGillivray B (2020) PLoS ONE 15(4), e0230416. 
speakers:
  - name: Kara Woo

>     bio: Kara Woo is a Principal Bioinformatics Engineer at [Sage Bionetworks](http://sagebionetworks.org/) where she leads a team of developers building tools and infrastructure for open science. She has previously worked as an information manager at Washington State University and at the National Center for Ecological Analysis and Synthesis (NCEAS), where she combined data management with fieldwork at a remote Siberian lake. Kara is a member of the rOpenSci Code of Conduct committee. Kara on [GitHub](https://github.com/karawoo), [Twitter](https://twitter.com/kara_woo), [Website](https://karawoo.com/)

Author: ```suggestion
    bio: Kara Woo is a Principal Bioinformatics Engineer at [Sage Bionetworks](http://sagebionetworks.org/) where she leads a team of developers building tools and infrastructure for open science. Kara is a member of the rOpenSci Code of Conduct committee. Kara on [GitHub](https://github.com/karawoo), [Twitter](https://twitter.com/kara_woo), [Website](https://karawoo.com/)
```

    img: kara-woo.jpg
  - name: Daniella Lowenberg

>     bio: Daniella Lowenberg is the Product Manager at [Dryad](https://datadryad.org/) and Principal Investigator of [Make Data Count](https://makedatacount.org/) within the California Digital Library at University of California. “she was a Publications Manager at PLOS ONE where she implemented and oversaw the PLOS Data Policy. For the last three years, Daniella has been a data publishing and data metrics product manager.” Daniella on [GitHub](https://github.com/dlowenberg), [Twitter](https://twitter.com/danilowenberg)

Author: ```suggestion
    bio: Daniella Lowenberg is the Product Manager at [Dryad](https://datadryad.org/) and Principal Investigator of [Make Data Count](https://makedatacount.org/) within the California Digital Library at University of California. Daniella on [GitHub](https://github.com/dlowenberg), [Twitter](https://twitter.com/danilowenberg)
```

    img: daniella-lowenberg.jpg
  - name: Matt Jones
    bio: Matt Jones is Director of Informatics R&D at the National Center for Ecological Analysis and Synthesis ([NCEAS](https://www.nceas.ucsb.edu/)), Principal Investigator of the [NSF Arctic Data Center](https://arcticdata.io/), and Director of [DataONE](https://www.dataone.org/) at University of California Santa Barbara. Matt on [GitHub](https://github.com/mbjones), [Twitter](https://twitter.com/metamattj), [Website](http://matt.magisa.org/)
    img: /img/team/matt-jones.jpg
  - name: Carl Boettiger

>     bio: Carl Boettiger is Assistant Professor in the [Department of Environmental Science, Policy and Management at UC Berkeley](http://ourenvironment.berkeley.edu/), a co-founder and leadership team member of rOpenSci. Carl on [GitHub](https://github.com/cboettig), [Twitter](https://twitter.com/cboettig), [Website](https://www.carlboettiger.info/)

Author: ```suggestion
    bio: Carl Boettiger is Assistant Professor in the [Department of Environmental Science, Policy and Management at UC Berkeley](http://ourenvironment.berkeley.edu/), a Co-founder and strategic advisor of rOpenSci. Carl on [GitHub](https://github.com/cboettig), [Twitter](https://twitter.com/cboettig), [Website](https://www.carlboettiger.info/)
```
